ui/src/app/styles/colors.ts

```diff
@@ -24,6 +24,8 @@ export default {
     notebook: '#8BC990'
   },
   white: '#fff',
+  purple: 'purple',
```

Comment:
We are avoiding adding new colors to this file to avoid file explosion. Purple or Blue should probably exist as one of our existing colors in our palette. (Maybe primary and accent is what you are looking for?)

If you're interested, I'm going to be giving rationale at the Eng Sync today.

Reply:
This was not an intentional change. I couldn't deploy the UI locally, so just added these to prevent compile errors till the fix came through. Removed.